### PR TITLE
[three] r160

### PR DESCRIPTION
**Pull request by Methuselah96**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/three-types/three-ts-types/issues/699
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by joshuaellis — APPROVED** [review]

File: types/three/tslint.json
Comment (peterblazejewicz) [inline types/three/tslint.json:1]: this file is no longer used, as all DT types has been migrated to ESLint:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67890
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67891
If you folks are using generated content, please consider updating your tooling, thx!

Comment (peterblazejewicz) [reply types/three/tslint.json:1]: otherwise lgmt

**Review by peterblazejewicz — APPROVED** [review]

File: types/three/tslint.json
Comment (Methuselah96) [reply types/three/tslint.json:1]: Nice catch! Will be fixed in the next release by https://github.com/three-types/three-ts-types/pull/726.

**Comment by Methuselah96** [comment]
Ready to merge

**Pull request merged**